src/libcommonserver/db/sqlitequery.cpp

```diff
@@ -198,20 +196,20 @@ bool SqliteQuery::nullValue(int index) const {
     return sqlite3_column_type(_stmt.get(), index) == SQLITE_NULL;
 }
 
-const std::string SqliteQuery::stringValue(int index) const {
+std::string SqliteQuery::stringValue(int index) const {
```

Comment:
```suggestion
std::string SqliteQuery::stringValue(const int index) const {
```
To please SonarCloud.